src/main/webapp/app/assessment/manage/list-of-complaints/list-of-complaints.component.html

```
@@ -1,4 +1,4 @@
-<div class="mt-n1 me-n1 course-info-bar">
+<div class="mt-n1 me-n1 course()-info-bar">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)